app/src/main/java/org/dhis2/Bindings/CategoryOptionExtensions.kt

```diff
@@ -9,3 +10,11 @@ fun CategoryOption.inDateRange(date: Date?): Boolean {
             (endDate() == null || date.before(endDate()))
     } ?: true
 }
+
+fun CategoryOption.inOrgUnit(orgUnitUid: String?): Boolean {
+    return orgUnitUid?.let {
+        organisationUnits()?.takeIf { it.isNotEmpty() }?.let {
+            orgUnitUid in UidsHelper.getUidsList(it)
+        } ?: true
+    } ?: true
+}
```

Comment:
Should this go to SDK logic @Balcan ?

Reply:
Yes, this could be helpful in the SDK. We have to tell Victor

Reply:
Does it makes sense to move it to the logic package?

Reply:
Okey, we could move the whole class